src/monitor.rs

```diff
@@ -121,6 +134,15 @@ impl MonitorHandle {
 
     /// Returns the top-left corner position of the monitor relative to the larger full
     /// screen area.
+    ///
+    /// ## Platform-specific
+    ///
+    /// **Web:** Always returns `{ x: 0, y: 0 }` without
```

Comment:
Maybe we should say that we return `::default`?

Reply:
Done.